benchmarks/benchmarks/stats_sampling.py

```diff
@@ -0,0 +1,123 @@
+import numpy as np
+from .common import Benchmark, safe_import
+
+with safe_import():
+    from scipy import stats
+with safe_import():
+    from scipy.stats._distr_params import distdiscrete
+
+
+# Beta distribution with a = 2, b = 3
+class contdist1:
```

Comment:
why did you define scalar functions for pdf etc? another option would be to use `lazywhere` for the pdf, dpdf and `scipy.special.btdtr` for the cdf.

Reply:
or is it enough to define a scalar function? (I need to check the documentation below)

Reply:
so it is clearly documented in the API, I suggest to point it out in the tutorial, too.

Reply:
> so it is clearly documented in the API, I suggest to point it out in the tutorial, too

Nice idea! I will add this point to the tutorial